--- a/lua/plugins.lua
+++ b/lua/plugins.lua
@@ -67,29 +67,39 @@ require("packer").startup(function()
   use 'p00f/nvim-ts-rainbow'
   use 'romgrk/nvim-treesitter-context'
   use 'phaazon/hop.nvim'
+  -- use 'mfussenegger/nvim-dap'
 end)
 
-vim.o.inccommand = 'nosplit'
-vim.o.hlsearch = true
-vim.o.ic = true
-vim.o.smartcase = true
-vim.o.number = true
-vim.o.relativenumber = true
-vim.o.scrolloff = 4
-vim.o.hidden = true
-vim.o.mouse = 'a'
+vim.opt.inccommand = 'nosplit'
+vim.opt.hlsearch = true
+vim.opt.ic = true
+vim.opt.smartcase = true
+vim.opt.number = true
+vim.opt.relativenumber = true
+vim.opt.scrolloff = 5
+vim.opt.hidden = true
+vim.opt.mouse = 'a'
 vim.opt.shortmess:append 'c'
-vim.o.updatetime = 250
-vim.o.splitright = true
-vim.o.splitbelow = true
-vim.o.swapfile = false
-vim.o.undofile = true
-vim.o.undodir = vim.fn.stdpath('cache')..'/undo'
-vim.o.shiftwidth = 2
-vim.o.softtabstop = 2
-vim.o.termguicolors = true
-vim.o.signcolumn = 'number'
-vim.o.guifont = 'FantasqueSansMono Nerd Font:h16'
+vim.opt.updatetime = 250
+vim.opt.timeoutlen = 500
+vim.opt.splitright = true
+vim.opt.splitbelow = true
+vim.o.switchbuf = 'useopen,uselast'
+vim.opt.swapfile = false
+vim.opt.undofile = true
+vim.opt.undodir = vim.fn.stdpath('cache')..'/undo'
+vim.opt.shiftwidth = 2
+vim.opt.softtabstop = 2
+vim.opt.expandtab = true
+vim.opt.shiftround = true
+vim.opt.showbreak = [[↪ ]]
+vim.opt.breakindentopt = 'sbr'
+vim.opt.linebreak = true
+vim.opt.termguicolors = true
+vim.opt.signcolumn = 'number'
+vim.opt.ruler = false
+vim.opt.autowriteall = true
+-- vim.opt.guifont = 'FantasqueSansMono Nerd Font:h16'
 
 -- colorscheme
 vim.g.everforest_enable_italic = 1
