--- a/lua/keymaps.lua
+++ b/lua/keymaps.lua
@@ -82,6 +82,7 @@ keymap('n', 'fp', ':lua require("telescope").extensions.neoclip.default()<cr>',
 
 -- neovim-session-manager
 keymap('n', 'fs', ':lua require("telescope").extensions.sessions.sessions()<cr>', opts)
+keymap('n', '<leader>ss', ':lua require("session_manager").save_session(filename)<cr>', nor)
 -- lsp-fastaction
 keymap('n', '<leader>a', '<cmd>lua require("lsp-fastaction").code_action()<CR>', opts)
 keymap('v', '<leader>a', '<esc><cmd>lua require("lsp-fastaction").range_code_action()<CR>', opts)
