--- a/lua/plugins.lua
+++ b/lua/plugins.lua
@@ -52,17 +52,16 @@ require("packer").startup(function()
   use {'instant-markdown/vim-instant-markdown'}
   use 'akinsho/nvim-toggleterm.lua'
   use 'voldikss/vim-translator'
-  use 'liuchengxu/vista.vim'
   use 'norcalli/nvim-colorizer.lua'
   use 'akinsho/dependency-assist.nvim'
   use { 'nvim-treesitter/nvim-treesitter', run = ':TSUpdate' }
   use 'ahmedkhalf/project.nvim'
   use 'AndrewRadev/splitjoin.vim'
   use 'AckslD/nvim-neoclip.lua'
   use 'p00f/nvim-ts-rainbow'
   use 'romgrk/nvim-treesitter-context'
   use 'phaazon/hop.nvim'
   use {'mfussenegger/nvim-dap', requires = 'rcarriga/nvim-dap-ui'}
 end)
 
 vim.opt.inccommand = 'nosplit'
