--- a/lua/plugins.lua
+++ b/lua/plugins.lua
@@ -57,14 +57,13 @@ require("packer").startup(function()
   use 'AndrewRadev/splitjoin.vim'
   use 'AckslD/nvim-neoclip.lua'
   use 'p00f/nvim-ts-rainbow'
-  use 'romgrk/nvim-treesitter-context'
   use 'phaazon/hop.nvim'
   use {'mfussenegger/nvim-dap', requires = 'rcarriga/nvim-dap-ui'}
   use 'simrat39/symbols-outline.nvim'
   use 'goolord/alpha-nvim'
 
   -- Bug
   -- use 'Shatur/neovim-session-manager'
 end)
 
 vim.opt.inccommand = 'nosplit'
